docs/ci.md

```diff
@@ -27,8 +27,22 @@ for building free-threaded wheels on all platforms. If your project releases
 nightly wheels, we suggest configuring `cibuildwheel` to build nightly
 free-threaded wheels.
 
-You will need to specify the following variables in the environment for the
-cibuildwheel action:
+If your project depends on Cython or the NumPy C API, you will need to install a
+Cython and/or NumPy nightly wheel in the build, as the newest stable release of
+Cython cannot generate code that will compile under the free-threaded build and
+the newest stable release of NumPy does not support free-threaded python. Cython
+3.1.0 and NumPy 2.1.0 will be the first stable releases to support free-threaded
+python.
+
+You can install nightly wheels for both Cython and NumPy using the following
+install command:
+
+```bash
+pip install -i https://pypi.anaconda.org/scientific-python-nightly-wheels/simple cython numpy
+```
+
+To ensure wheels are built correctly under cibuildwheel, you will need to
+specify the following variables in the environment for the cibuildwheel action:
```

Comment:
Can we include a link to https://cibuildwheel.pypa.io/en/stable/options/#build-frontend somewhere here so that people can see that both `build` and `pip` are possible options?